tests/unit/test_dimension.py

```diff
@@ -1208,14 +1208,14 @@ def it_knows_the_ordered_element_ids_for_an_explicit_sort(
         assert _OrderSpec(None, None).element_ids == expected_value
 
     def it_knows_the_sort_vector_insertion_id(self, _order_dict_prop_):
-        _order_dict_prop_.return_value = {"insertion_id": 42}
+        _order_dict_prop_.return_value = {"id": 42}
```

Comment:
Ah, no, I think I see now. So the _insertion_ gets `"id":`, but the _order_ still gets `"insertion_id":`. So the id of the insertion itself appears in `transforms.rows_dimension.insertions[i].id` but the _reference_ to it in the order dict appears at `order.insertion_id`.